src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -45,6 +45,22 @@ class Preview extends React.Component<Props> {
     typeDisposer: () => mixed;
     dataDisposer: () => mixed;
 
+    @computed get webspaceKey() {
+        const {
+            router: {
+                attributes: {
+                    webspace,
```

Comment:
can we define a type for this `?string` destructering here, so flow would fail when try to call function which do not accept null or undefined

Reply:
That's a good idea, will fix that. Will probably include some other changes as well, therefore going to do that in a separate PR in release/2.0.